acceptance_tests/features/pages/sign_in_internal.py

```diff
@@ -22,5 +22,17 @@ def enter_incorrect_password():
     browser.driver.find_element_by_id('password').send_keys('words')
 
 
-def internal_sign_in_btn():
+def internal_sign_in_button():
     browser.find_by_id('SIGN_IN_BUTTON').click()
+
+
+def username_required():
+    browser.find_link_by_text('Username is required')
+
+
+def password_required():
+    browser.find_link_by_text('Password is required')
+
+
+def authentication_error_message():
```

Comment:
I think these functions should be things like 'check_authentication_error_message', 'check_username_required'.  Each of these functions are verbs and are much easier to understand and remember if they're named as much.